demos/collection/jsactionsgrid.php

```diff
@@ -10,7 +10,6 @@
 // Demo for Model action in Grid
 
 $country = new CountryLock($app->db);
-$country->tryLoadAny();
```

Comment:
@ibelar is this (and the same in `demos/layout/layout-panel.php`) load needed?

Reply:
Sure is. The model record needs to be loaded in order to use certains model action.

Reply:
> Sure is. The model record needs to be loaded in order to use certains model action.

My bad. I have read too fast the first time and thought you were refering another page. 😄 Yes, it can be remove in both files.

Reply:
thanks